monosize.config.mjs

```diff
@@ -8,7 +8,9 @@ import createAzureStorage from 'monosize-storage-azure';
 const config = {
   repository: 'https://github.com/microsoft/fluentui',
   storage: createAzureStorage({
-    endpoint: 'https://fluentbundlesize.azurewebsites.net/api/latest',
+    authType: 'AzurePipelinesCredential',
+    endpoint: 'https://fluent-bundlesize.azurewebsites.net/api/fluentuilatest',
+    tableName: 'fluentuilatest',
```

Comment:
Note: I created a new azure storage and function app that bundle size testing will use going forward